manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml

```diff
@@ -9,6 +9,10 @@ Installers:
   InstallerType: inno
   InstallerUrl: https://download.advanced-ip-scanner.com/download/files/Advanced_IP_Scanner_2.5.3850.exe
   InstallerSha256: 87BFB05057F215659CC801750118900145F8A22FA93AC4C6E1BFD81AA98B0A55
+  InstallerSwitches: 
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
+    Log: /LOG="<LOGPATH>"
```

Comment:
These are also redundant when the InstallerType is set to Nullsoft, Inno, Burn, Msi, Msix, Appx, or Wix :)
I believe this https://github.com/microsoft/winget-cli/blob/b79e4087821c993f83f7c4731bbde7aa25612c73/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L449-L481 is what defines that